compositional/1D_COMP_NO_WELLS.DATA

```diff
@@ -0,0 +1,214 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2024, SINTEF Digital, TNO
+
+-- This is a 1D, pressure-driven CO₂ flooding example involving three components and two phases.
+-- The components are CO₂, methane, and decane.
+-- The first and last cells are assigned very large pore volumes to emulate a source and sink,
+-- respectively, with constant pressure and compositions.
+
+------------------------------------------------------------------------
+RUNSPEC
+------------------------------------------------------------------------
+TITLE
+   SIMPLE CO2
+
+METRIC
+
+TABDIMS
+/
+OIL
+GAS
+DIMENS
+30 1 1
+/
+
+COMPS
+3 /
+
+-- FULLIMP /
+
+WELLDIMS
+2 2 1* 2/
+
+START
+  1 'JAN' 2016  /
+
+REGDIMS
+1 1 0 0 /
+
+UNIFOUT
+------------------------------------------------------------------------
+GRID
+------------------------------------------------------------------------
+INIT
+DX
+30*10
+/
+DY
+30*30
+/
+DZ
+30*1
+/
```

Comment:
I generally prefer the tensor product form for this, i.e.
```
DXV
  30*10
/
DYV
  10
/
DZV
 1
/
```
since that's less ambiguous.